pkg/apis/vpc/v1alpha1/condition_types.go

```diff
@@ -12,6 +12,7 @@ const (
 	GatewayConnectionReady     ConditionType = "GatewayConnectionReady"
 	AutoSnatEnabled            ConditionType = "AutoSnatEnabled"
 	ExternalIPBlocksConfigured ConditionType = "ExternalIPBlocksConfigured"
+	DeleteFailure              ConditionType = "DeletionFailed"
```

Comment:
We added crd doc generation script recently, could you run `make generate-api-docs` to include the change here in doc?